src/DataExplorer/DataExplorerVisualizations.jsx

```diff
@@ -18,6 +18,14 @@ class DataExplorerVisualizations extends React.Component {
     super(props);
     this.state = {
       manifestEntryCount: 0,
+      exportInProgress: false,
+      exportFileName: null,
+      toasterOpen: false,
+      exportStatus: null,
```

Comment:
I think we can get rid of this since we have `exportErrorMsg` too. So if it returns 200, just set `exportErrorMsg` to null, if it returns an error set this value to something

and then assume that a null msg means 200 and one with value means there was an error?